Example/ViewController+MiniApp.swift

```diff
@@ -45,7 +45,6 @@ extension ViewController {
                         self.fetchMiniAppUsingId(title: NSLocalizedString("error_title", comment: ""), message: NSLocalizedString("error_single_message", comment: ""))
                     }
                 }
-                self.dismissProgressIndicator()
```

Comment:
I don't understand why this is blocking load by mini-app functionality? I can still load mini-app by ID

Reply:
MiniApp is loaded, controller is presented and dismissed even before it appeared, because e `dismissProgressIndicator` only does a UIViewController `self.dismiss` method call which dismisses the presented controller.